README.md

```diff
@@ -118,6 +118,28 @@ In this example, the Action will load the OIDC token from the GitHub-provided en
 ```
 In this example, the secret `AWS_ROLE_TO_ASSUME` contains a string like `arn:aws:iam::123456789100:role/my-github-actions-role`.  To assume a role in the same account as the static credentials, you can simply specify the role name, like `role-to-assume: my-github-actions-role`.
 
+```yaml
+    - name: Configure AWS Credentials
+      id: aws-credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-region: us-east-2
+        role-to-assume: arn:aws:iam::123456789100:role/my-github-actions-role
+        role-session-name: MySessionName
+        role-output-credentials: true
+    
+    - name: Get Caller Identity
+      env:
+        AWS_REGION: '${{ steps.aws-credentials.outputs.aws-region }}'
+        AWS_DEFAULT_REGION: '${{ steps.aws-credentials.outputs.aws-default-region }}'
+        AWS_ACCESS_KEY_ID: '${{ steps.aws-credentials.outputs.aws-access-key-id }}'
+        AWS_SECRET_ACCESS_KEY: '${{ aws-credentials.aws.outputs.aws-secret-access-key }}'
```

Comment:
I think `AWS_SECRET_ACCESS_KEY` is missing `steps`.

Shouldn't it be: `AWS_SECRET_ACCESS_KEY: '${{ steps.aws-credentials.aws.outputs.aws-secret-access-key }}'`?

Reply:
You are absolutely right